Makefile

```diff
@@ -211,10 +211,10 @@ TEST_TARGETS := test-unit test-unit-amino test-unit-proto test-ledger-mock test-
 # a new rule, customise ARGS or TEST_PACKAGES ad libitum, and
 # append the new rule to the TEST_TARGETS list.
 
-test-unit: ARGS=-tags='cgo ledger test_ledger_mock'
-test-unit-amino: ARGS=-tags='ledger test_ledger_mock test_amino'
-test-ledger: ARGS=-tags='cgo ledger'
-test-ledger-mock: ARGS=-tags='ledger test_ledger_mock'
+test-unit: ARGS=-tags='cgo ledger test_ledger_mock norace'
+test-unit-amino: ARGS=-tags='ledger test_ledger_mock test_amino norace'
+test-ledger: ARGS=-tags='cgo ledger norace'
```

Comment:
why leger has `norace`? For me, leger souns like a target with many concurrent processes.

Reply:
The `ledger` tag enables ledger support, the `test_ledger_mock` enables ledger device simulation.
By appending the `norace` tag, I'm signalling that the tests are not running with the test detector in place, thus all can run.

Reply:
By thew way, the `test-race` rule runs ledger tests with `-race` enabled successfully:

```
test-race: ARGS=-race -tags='cgo ledger test_ledger_mock'
```

Reply:
ah, ledger -> the wallet? Could you add a comment please?

Reply:
Can't follow, sorry. What should the comment be about?

Reply:
about `ledger`

Reply:
Sorry, what's unclear about ledger? We've historically called the functionality as `ledger support`. The `test-ledger` rule has been there since ever, developers know that that is to test real `ledger support`. How could a comment clarify this further? IMvHO stating the obvious adds no value

Reply:
When I first looked at it, I was thinking that we are testing with a real blockchain structure (ledger = blockchain), a 3rd party company product.